### DISCUSSION TITLE
未连接到守护进程，大部分功能不可用

### DISCUSSION THREAD

**Discussion by Antis62** [natfrp/wiki#212] [使用问题] [answered]
用了三天没问题，今天一登录就显示 未连接到守护进程，大部分功能不可用，然后换了一堆节点，一个都连不上，显示如下
![LW(HCDMKE}YSX_X8O {)@4F](https://user-images.githubusercontent.com/118428304/202383564-8241ce73-1217-4253-a57e-5f743c4d6c7f.png)

**Comment by Antis62** [comment]
抱歉，放错图了
![1668670681774](https://user-images.githubusercontent.com/118428304/202385055-8a5e806f-3bf4-4f55-b008-2964c629b93d.jpg)
这张的报错

**Reply by Antis62** [reply]
已经试过以管理员身份运行了，还是一样的结果

**Comment by fakeboboliu** [comment, marked as answer]
此问题通常来源于您的网络，

如果您在使用校园网/企业网，网络管理员可能会根据安全要求屏蔽 FRP 协议。
这种屏蔽是通常是基于相关安全规定做出的决定，我们建议您不要尝试使用本服务和/或其他内网穿透服务，以免背负安全责任。

如果您知道自己在做什么，请选择一个枣庄外的节点，然后在选项中打开「强制使能 TLS」，
请注意，这个选项可能会使您绕过安全防火墙，并造成安全漏洞，
在使用前请确认您认为 [安全指南](https://doc.natfrp.com/#/bestpractice/security) 的所有内容都已被完美实施。
或者如果您认为安全是个屁，不会有任何人来向您找茬，并确认在出现任何问题与我们的服务无关，您也可以尝试此条建议。

**Reply by Antis62** [reply]
我的确是使用校园网，正常使用了几天之后今天突然用不了了，看样子似乎确实是学校网管出手了。你的意思是这个「强制使能 TLS」是被大部分校园网/企业网禁止的吗？

**Reply by fakeboboliu** [reply]
> 你的意思是这个「强制使能 TLS」是被大部分校园网/企业网禁止的吗？

打开此选项可能可以绕过大部分网管做出的限制，但是请注意，使用内网穿透类工具造成的安全隐患仍然存在。

**Reply by Antis62** [reply]
> > 你的意思是这个「强制使能 TLS」是被大部分校园网/企业网禁止的吗？
> 
> 打开此选项可能可以绕过大部分网管做出的限制，但是请注意，使用内网穿透类工具造成的安全隐患仍然存在。

是网络信息的安全隐患，还是人身安全隐患😂

**Reply by fakeboboliu** [reply]
> 是网络信息的安全隐患，还是人身安全隐患😂

显然，首先是网络安全隐患，如果网络安全隐患被发现或者被利用，那就是人身安全隐患了
